packages/api/endpoints/granules.js

```diff
@@ -549,14 +549,15 @@ async function bulkOperations(req, res) {
       type: 'BULK_GRANULE',
       envVars: {
         ES_HOST: process.env.ES_HOST,
-        GranulesTable: process.env.GranulesTable,
         granule_sns_topic_arn: process.env.granule_sns_topic_arn,
-        system_bucket: process.env.system_bucket,
-        stackName: process.env.stackName,
+        GranulesTable: process.env.GranulesTable,
         invoke: process.env.invoke,
+        KNEX_DEBUG: payload.knexDebug.toLowerCase() === 'true' || false,
```

Comment:
This will fail if `payload.knexDebug` is undefined. Are we guaranteeing that it'll be present somewhere?